tee-worker/client-api/sidechain-api/dist/package.json

```diff
@@ -7,12 +7,13 @@
 		"clean": "rm -rf dist build node_modules",
 		"update-metadata": "../../bin/litentry-cli print-sgx-metadata-raw > build/litentry-sidechain-metadata.json",
 		"prepare-dir": "mkdir -p build && cp -rf prepare-build/* build",
-		"generate-from-defs": "pnpm tsx node_modules/@polkadot/typegen/scripts/polkadot-types-from-defs.mjs  --package sidechain-api/interfaces --input build/interfaces --endpoint build/litentry-sidechain-metadata.json",
-		"generate-from-chain": "pnpm tsx node_modules/@polkadot/typegen/scripts/polkadot-types-from-chain.mjs --package sidechain-api/interfaces --output build/interfaces --endpoint build/litentry-sidechain-metadata.json --strict",
+		"generate-from-defs": "node --experimental-specifier-resolution=node --loader ts-node/esm node_modules/@polkadot/typegen/scripts/polkadot-types-from-defs.mjs --package sidechain-api/interfaces --input build/interfaces --endpoint build/litentry-sidechain-metadata.json",
+		"generate-from-chain": "node --experimental-specifier-resolution=node --loader ts-node/esm node_modules/@polkadot/typegen/scripts/polkadot-types-from-chain.mjs --package sidechain-api/interfaces --output build/interfaces --endpoint build/litentry-sidechain-metadata.json --strict",
```

Comment:
Now I'm confused; are we gonna run these with `node`, `tsx`, or `ts-node`? Do we need the `ts-node` dependency at all? :thinking: 

(also, should probably invoke it as `pmpm exec node` to make sure it's using the right environment)

Reply:
We can add `ppm exec`.According to typegen's [latest docs](https://github.com/polkadot-js/docs/blob/22312dfd76d2e4445962ea54857683e732461a7a/docs/api/examples/promise/typegen.md), we're better off using node to run. Anyway, after testing, `ts-node` does not work.